components/AutomateBettingForm.tsx

```diff
@@ -128,28 +142,12 @@ export default function SpreadsheetForm() {
         setUserData(updatedUserData);
     }
 
-    useEffect(() => {
-        console.log("Getting stored data")
-        const storedUserData = JSON.parse(window.localStorage.getItem('user-data'));
-        const storedtableData = JSON.parse(window.localStorage.getItem('processed-data'));
-        const storedApiKey = window.localStorage.getItem('api-key');
-        if (storedApiKey) {
-            setApiKey(storedApiKey);
-        }
-    
-        if (storedUserData) {
-            setUserData(storedUserData);
-        } else if (storedtableData) {
-            setProcessedData(storedtableData);
-        }
-    }, []);
```

Comment:
this seemed to be nearly the same as the useEffect above (only difference was in logging) so I removed it